examples/all_examples.jl

```diff
@@ -1 +1 @@
-const EXAMPLES = String[]
+const EXAMPLES = String["../docs/src/examples/gol_lazar_belta.jl"]
```

Comment:
Replace this by `"gol_lazar_belta.jl"` and change `const EXAMPLES_DIR = joinpath(@__DIR__, "..", "examples")` to `const EXAMPLES_DIR = joinpath(@__DIR__, "src", "examples")`.
This should fix the error in CI.

Reply:
Currently, the error comes from:
```julia
julia> split("../docs/src/examples/gol_lazar_belta.jl", ".")
4-element Array{SubString{String},1}:
 ""
 ""
 "/docs/src/examples/gol_lazar_belta"
 "jl"
```